android/capacitor/src/main/java/com/getcapacitor/BridgeActivity.java

```diff
@@ -167,6 +175,8 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
 
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
```

Comment:
Try to avoid adding unrelated changes to PRs. It makes it difficult to review and creates a less meaningful history. We should strive for single-purpose PRs with small scopes.